src/v0/destinations/facebook_pixel/networkHandler.js

```diff
@@ -85,7 +142,12 @@ const getStatus = (error) => {
     errorStatus = 429;
   }
 
-  return errorStatus;
+  let errorMessage = errorDetail?.messageDetails?.message;
+  if (errorDetail?.messageDetails?.field) {
```

Comment:
What's the idea behind `field` parameter ?

Reply:
The parameter field is related to error message.
We will extract whatever message is available in the field
For example
If the response from facebook_pixel looks like this
```
error": {
    "message": "Invalid parameter",
    "type": "OAuthException",
    "code": 100,
    "error_subcode": 2804007,
    "is_transient": false,
    "error_user_title": "Invalid IP Address",
    "error_user_msg": "The IP address you entered is invalid. Enter a valid IP address.",
    "fbtrace_id": "AZN7qSAksRz48Xdc_YNQ9_7"
}

Here we can see clearly that if we use the content in `error_user_title` field, that should sufficiently provide us context about the error message itself
```
Since field is related to message, I thought it is best to put in `messageDetails` object, making it easy to understand.